src/plugins/mer/merrunconfigurationaspect.cpp

```diff
@@ -129,7 +134,13 @@ class MerRunConfigWidget : public QWidget
                 MerQmlLiveBenchManager::offerToStartBenchIfNotRunning();
         });
 
-        layout()->addWidget(qmlLiveWidget);
+        qmlLiveLayout->addWidget(qmlLiveWidget);
+        outerLayout->addLayout(qmlLiveLayout);
+
+        QCheckBox* debuggerEnvironmentCheckbox = new QCheckBox(tr("Bypass OPENSSL ARM capability detection while debugging"));
```

Comment:
```suggestion
        QCheckBox* debuggerEnvironmentCheckbox = new QCheckBox(tr("Bypass OpenSSL ARM capability detection while debugging"));
        debuggerEnvironmentCheckbox->setToolTip(tr("When checked, \"OPENSSL_armcap=1\" (ARMV7_NEON) "
                "will be set in the process environment under debugger. "
                "The way OpenSSL detects CPU capabilities would lead to receiving SIGILL at startup."));
```